assembly-machine-war/src/main/webapp/WEB-INF/classes/codenvy/che-machine-configuration.properties

```diff
@@ -34,6 +34,11 @@ vfs.local.directory_mapping_file=${catalina.base}/temp/vfs
 #this path is relative to user home directory
 che.workspace.metadata = che/.workspace
 
+# path to templates' description folder
```

Comment:
We use it here https://github.com/codenvy/che-core/blob/f2910740fa999894b2a264258e32dc94024be33a/platform-api/che-core-api-project/src/main/java/org/eclipse/che/api/project/server/ProjectTemplateDescriptionLoader.java#L46-L61
project.template_descriptions_dir and project.template_location_dir are different entities though bouth refer to the same place